imports/ui/case/case.mui-styles.js

```diff
@@ -32,7 +32,8 @@ export const infoIconStyle = {
 }
 
 export const addPersonIconStyle = {
-  color: '#a4a4a4',
+  color: 'var(--bondi-blue)',
```

Comment:
Instead of just affecting the color on the new "invite" link button, it also affects the existing button on the details view.
Please see the screenshots attached to the next issue comment for explanation.

You should use a different type definition for each button, or find another way to differentiate them